app/common/patient/directives/scheduleList.js

```diff
@@ -0,0 +1,67 @@
+(function () {
+  'use strict';
+
+  angular
+    .module('common.patient')
+    .directive('scheduleList', scheduleList);
+
+  scheduleList.$inject = [];
+
+  /* @ngInject */
+  function scheduleList() {
+    var directive = {
+      bindToController: true,
+      controller: ScheduleListController,
+      controllerAs: 'vm',
+      restrict: 'E',
+      scope: {
+        onPatientSelect: '&'
+      },
+      templateUrl: '../common/patient/directives/scheduleList.html'
+    };
+    return directive;
+  }
+
+  ScheduleListController.$inject = ['$q', 'commonService', 'cohortService', 'observationsService', 'visitService'];
+
+  /* @ngInject */
+  function ScheduleListController($q, commonService, cohortService, observationsService, visitService) {
+
+    var vm = this;
+
+    vm.patients = [];
+
+    activate();
+
+    ////////////////
+
+    function activate() {
+      cohortService.getMarkedForConsultationToday().then(function (members) {
```

Comment:
Schedule list uses different cohorts for each module:
- Registration: Patients with next consultation date equal to today
- Clinic: Patients with next consultation date equal to today and checked in today
- Pharmacy Patients with next pickup date equal to today

This directive contain only the first

Reply:
I could not see this in the previous implementation could you show me where to find it?

Reply:
The difference is the cohort definition for each case: [clinic](https://github.com/esaude/esaude-emr-poc/blob/master/app/clinic/controllers/scheduleListController.js#L11), [registration](https://github.com/esaude/esaude-emr-poc/blob/master/app/registration/controllers/scheduleListController.js#L13) and [pharmacy](https://github.com/esaude/esaude-emr-poc/blob/master/app/pharmacy/controllers/scheduledPatientsController.js#L7)